app/presenters/two_factor_auth_code/webauthn_authentication_presenter.rb

```diff
@@ -16,6 +16,14 @@ def initialize(data:, view:, service_provider:, remember_device_default: true,
       )
     end
 
+    def webauthn_title
```

Comment:
Can we add some test coverage for the two branches of this method in `webauthn_authentication_presenter_spec.rb` ?

Reply:
Yes. That's added.